models/openxswap/optimism/openxswap_optimism_schema.yml

```
@@ -0,0 +1,80 @@
+version: 2
+
+models:
+  - name: openxswap_optimism_trades
+    meta:
+      blockchain: optimism
+      sector: dex
+      project: openxswap
+      contributors: ARDev097
+    config:
+      tags: ['optimism','openxswap','trades','dex']
+    description: >
+        openxswap contract trades on optimism 
+    columns:
+      - &blockchain
```

We can apply uniqueness and check_dex_seed here:
```suggestion
    description: >
        openxswap contract trades on optimism 
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - block_date
            - blockchain
            - project
            - version
            - tx_hash
            - evt_index
      - check_dex_seed:
          blockchain: optimism
          project: openxswap
          version: 1
    columns:
      - &blockchain
```